gluten-ut/common/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -172,7 +171,35 @@ trait GlutenTestsTrait extends GlutenTestsCommonTrait {
     } else {
       logInfo("Has unsupported data type, fall back to vanilla spark.\n")
     }
-    checkResult(result, expected, expression)
```

Comment:
@loneylee Thanks for your fix! Can we separate this part of fix to another PR? Because we found some incorrect cases were not reported out previously, and we need to enhance the result comparison.

BTW, after this change, I found some Velox incorrectness was still not reported out by CI, so we need to improve it further.

Reply:
Here is new separated PR.https://github.com/oap-project/gluten/pull/1465

Reply:
Thanks!